login/login.c

```diff
@@ -291,6 +346,11 @@ int login_run(glome_login_config_t* config, const char** error_tag) {
     return EXITCODE_PANIC;
   }
 
+  if (config->options & VERBOSE) {
+    login_syslog(config, pamh, LOG_DEBUG, "host ID: %s", host_id);
+    login_syslog(config, pamh, LOG_DEBUG, "action: %s", action);
```

Comment:
Why not write both in the same syslog call?

Reply:
Done.